### ISSUE TITLE
favicon.ico forbidden

### ISSUE THREAD

**Issue by 15773200610** [ctripcorp/apollo#2892]
Apollo multi-environment deployment, the adminservice and configservice of each environment have been started, and their corresponding euroka service registration addresses have also been seen and started normally. But I can't board it when I log in to the portal, report to /favicon.ico forbidden.
What is the problem with you guys?

**Comment by nobodyiam** [comment]
`favicon.ico forbidden` is not the root cause, you may check your username/password and portal log for more information.

**Comment by 15773200610** [comment]
Thank you for your reply. I checked afterwards and found that no portal logs were generated and the username and password were correct. I suddenly thought that the default configuration of the database had been modified due to the case sensitivity of mysql. So I did a test: first delete the database, then turn off the case sensitivity, re-import the database, restart and find that it can log in. Later, I deleted the database, opened the case sensitivity, and then re-imported the database. After restarting, the same problem occurred. So the problem was found, but the solution was still clueless. Would like to ask everyone.

**Comment by 15773200610** [comment]
> `favicon.ico forbidden` is not the root cause, you may check your username/password and portal log for more information.

Thank you for your reply. I checked afterwards and found that no portal logs were generated and the username and password were correct. I suddenly thought that the default configuration of the database had been modified due to the case sensitivity of mysql. So I did a test: first delete the database, then turn off the case sensitivity, re-import the database, restart and find that it can log in. Later, I deleted the database, opened the case sensitivity, and then re-imported the database. After restarting, the same problem occurred. So the problem was found, but the solution was still clueless.

**Comment by 15773200610** [comment]
> `favicon.ico forbidden` is not the root cause, you may check your username/password and portal log for more information.

Thank you for your reply. I checked afterwards and found that no portal logs were generated and the username and password were correct. I suddenly thought that the default configuration of the database had been modified due to the case sensitivity of mysql. So I did a test: first delete the database, then turn off the case sensitivity, re-import the database, restart and find that it can log in. Later, I deleted the database, opened the case sensitivity, and then re-imported the database. After restarting, the same problem occurred. Want to ask what configuration needs to be done in case of case sensitivity?
